### ISSUE TITLE
Force browsers to fetch up-to-date resources

### ISSUE THREAD

**Issue by pafcu** [friendica/red#827]
Due to caching browsers sometimes have an old version of some resource (scripts, stylesheets) in their cache. This leads to strange behaviour.

A commonly accepted way to avoid this is to add something like "?version=XXX" (where XXX is updated as needed) to javascript and css when requesting them. XXX could be e.g. RED_VERSION as defined in boot.php, and the modification of the file name could be added to head_add_js and head_add_css in include/plugin.php.

Basically, to fix this, you'd need to change two lines slightly in include/plugin.php, and remember to update the version in version.inc whenever there's a change that requires a browser refresh.

**Comment by jeroenpraat** [comment]
This issue was moved to redmatrix/redmatrix#167
